users/views.py

```diff
@@ -269,9 +288,133 @@ def patch(self, request):
             return JsonResponse({'MESSAGE': 'PASSWORD_CHANGED'}, status=200)
         except json.decoder.JSONDecodeError:
             return JsonResponse({'MESSAGE': 'JSON_DECODE_ERROR'}, status=400)
-        except KeyError:
+        except KeyError:    
             return JsonResponse({'MESSAGE': 'KEY_ERROR'}, status=400)
         except TypeError:
             return JsonResponse({'MESSAGE': 'TYPE_ERROR'}, status=400)
         except ValueError:
             return JsonResponse({'MESSAGE': 'VALUE_ERROR'}, status=400)
+
+class ProfileListView(View):
+    
+    @query_debugger
+    def get(self, request):
+        sort_method = request.GET.get('sorted_by') if request.GET.get('sorted_by') else 'id'
```

Comment:
```suggestion
        sort_method = request.GET.get('sorted_by', 'id')
```
get() 메소드의 특성을 활용하여 'sorted_by' 를 못찾았을 시, default 로 'id'를 주실 수 있겠네요.

Reply:
피드백 감사합니다!! get을 제대로 활용하지 못했었네요